dev/TeachingTip/TeachingTip.cpp

```diff
@@ -121,6 +123,9 @@ void TeachingTip::OnApplyTemplate()
     OnIconSourceChanged();
     OnHeroContentPlacementChanged();
 
+    UpdateButtonAutomationProperties(m_actionButton.get(), m_actionButtonContent.get().Content());
```

Comment:
You don't need to add these new template parts. You can just use ActionButtonContent() and CloseButtonContent() properties.